src/assets/text/droughtNarrations_desktop.js

```diff
@@ -5,54 +5,108 @@ export default {
         title: 'Dust Bowl (1930 - 1941)',
         start_date: '1930-01-01', // Date marking start of annotation. Determines y trigger start position on deskop and order
+        end_date: '1937-08-31', // Date marking end of annotation. Determines y trigger end position on deskop
+        text: 'Drought hit most of the U.S. in the 1930s, but the farmers and ranchers of the Great Plains were hit hardest. Not enough rain combined with too much farming and grazing made the top layer of soil loose and dry. Heavy winds blew the dry soil into massive dust storms, which is why we call this period in history the “dust bowl.” </a>', // Narrative text
+        quote: 'Dust to eat, and dust to breathe and dust to drink. Dust in the beds and in the flour bin, on dishes and walls and windows, in hair and eyes and ears and teeth and throats, to say nothing of the heaped up accumulation on floors and window sills after one of the bad days.',
+        quote_source: '<a href="https://americanexperience.si.edu/historical-eras/the-great-depression-wwii/pair-dust-bowl-valley-farms/" target="_blank">Letter from Caroline Henderson to Secretary of Agriculture Henry A. Wallace, 1935</a>'
+      },
+      {
+        id: '1930b', // Unique id for annotation - order and # value don't matter - annotations are later sorted by start_date
+        title: 'Dust Bowl (1930 - 1941)',
+        start_date: '1937-08-31', // Date marking start of annotation. Determines y trigger start position on deskop and order
         end_date: '1943-08-31', // Date marking end of annotation. Determines y trigger end position on deskop
-        text: 'Drought hit most of the U.S. in the 1930s, but the farmers and ranchers of the Great Plains were hit hardest. Not enough rain combined with too much farming and grazing made the top layer of soil loose and dry. Heavy winds blew the dry soil into massive dust storms, which is why we call this period in history the “dust bowl.” The drought ended in 1941, but the lessons learned from the dust bowl continue to shape the farming and grazing practices we use today. </a>', // Narrative text
+        text: 'The drought ended in 1941, but the lessons learned from the dust bowl continue to shape the farming and grazing practices we use today. </a>', // Narrative text
         quote: 'Dust to eat, and dust to breathe and dust to drink. Dust in the beds and in the flour bin, on dishes and walls and windows, in hair and eyes and ears and teeth and throats, to say nothing of the heaped up accumulation on floors and window sills after one of the bad days.',
         quote_source: '<a href="https://americanexperience.si.edu/historical-eras/the-great-depression-wwii/pair-dust-bowl-valley-farms/" target="_blank">Letter from Caroline Henderson to Secretary of Agriculture Henry A. Wallace, 1935</a>'
       },
       {
-        id: '1952',
+        id: '1952a',
```

Comment:
The scroll triggers feel slightly off here:
The button highlights and text changes to 1950s as soon as in 1950, quote and fact appear:
<img width="1761" alt="image" src="https://github.com/DOI-USGS/drought-timeline/assets/17803537/74b37bcd-de49-4c36-9b0c-a0dd9975a248">

but then when we get to 1952 when the drought event was, only the text on the right changes. the fact and quote match with this anecdote, may want to have them appear here instead, or have all the irghthand text appear at the same time.
<img width="1761" alt="image" src="https://github.com/DOI-USGS/drought-timeline/assets/17803537/f44c047b-1696-4878-a154-8f92623224a3">

Reply:
- [ ] reduce the number of rows this annotation spans

Reply:
Because laptop screens are so small, the narration has to be split into two, but if I also animate map and/or quote at that same change time, then the changing narration gets lost. I purposely made on the the narration change so that it is apparent it's changing and it catches the eye. (Will ask HCD to test this approach on the laptop screen)

Reply:
![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/9394dcb1-b431-40c9-a36d-e63fc04340af)